lib/manager/cdn/update.ts

```diff
@@ -0,0 +1,28 @@
+import { logger } from '../../logger';
+import { Upgrade } from '../common';
+
+export function updateDependency(
+  fileContent: string,
+  upgrade: Upgrade
+): string | null {
+  const { depName, currentValue, newValue, managerData } = upgrade;
+  const { fileReplacePosition } = managerData;
+  const leftPart = fileContent.slice(0, fileReplacePosition);
+  const rightPart = fileContent.slice(fileReplacePosition);
+  const versionClosePosition = rightPart.indexOf('/');
+  const restPart = rightPart.slice(versionClosePosition);
+  const versionPart = rightPart.slice(0, versionClosePosition);
+  const version = versionPart.trim();
+  if (version === newValue) {
+    return fileContent;
+  }
```

Comment:
What's the possible scenario where this could occur?

Reply:
Should happen whenever the file has been updated already in a previous run